jobs/test-override.Jenkinsfile

```diff
@@ -78,6 +86,12 @@ try {
     }
     currentBuild.description = "[${descPrefix}] Running"
 
+    // resolve bodhi IDs to koji builds
+    for (id in bodhi_update_ids) {
+        def build_ids = shwrapCapture("curl -sSL https://bodhi.fedoraproject.org/updates/${id} | jq -r .update.builds[].nvr")
+        koji_build_ids += build_ids.split() as List
```

Comment:
should we check and error here if somehow the build_ids list is empty (shouldn't happen)? I'm OK either way.